src/TransitFunctionAppTest/FakeDeviceClient.cs

```diff
@@ -99,9 +101,11 @@ public Task<dynamic> GetDynamicDigitalTwinAsync()
             return Task<dynamic>.Factory.StartNew(() => fakeTwin.ToExpandoObject());
         }
 
-        public Task RegisterDesiredPropertyUpdateCallbackAsync(DesiredPropertyUpdateCallback methodHandler)
+        // collects the various handlers
+        public async void RegisterDesiredPropertyUpdateCallbackAsync(DesiredPropertyUpdateCallback callbackHandler)
```

Comment:
* async with not await ?
* same as above -async void-, how would the caller monitor / control it ?